packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -51,6 +51,8 @@ export const messages = {
   parameter_gatherer_enter_username_name: 'Enter target username',
   parameter_gatherer_enter_alias_name:
     'Enter an org alias or use default alias',
+  parameter_gatherer_enter_scratch_org_expiration_days:
+    'Enter the expiration days for the scratch org',
```

Comment:
Maybe 'Enter the number of days until scratch org expiration (1–30 days)'?

Reply:
"Enter the number of days until scratch org expiration (1-30 days)" sounds a little better in my opinion. What do you think? Also, does it make sense to check if the user inputted a valid number (1 to 30) instead of letting the command fail?

Reply:
@renatoliveira @ruthemmanuelle 
I like message but how about we put the default in there? Something like "**Enter the number of days until scratch org expiration (1-30 days, default 7 days)**". @ruthemmanuelle, just to check, is 30 days the max life?

For the user input, it should default to 7 so the user doesn't have to type anything unless they care to change the default. As for verification, yes, it should definitely verify the input is a number and it falls within the min/max range.

Reply:
Yes, 30 days is the max life for scratch orgs. If we're going to allow the user to leave the field blank to accept the default, how about this? 

'Enter the number of days until scratch org expiration (1–30), or press Enter to accept the 7-day default'

Note that there's an en dash, not a hyphen, in "1–30".

And, I agree that we should validate the user's input.

Reply:
Or we can also set the default seven day message as the placeholder :)

"Enter the number of days until scratch org expiration (1-30 days)"

placeholder: "Press Enter to accept the default (7 days)"

Reply:
Good idea re: making use of the placeholder so we can make this message shorter. 

For the org alias prompt, we use the default value as the placeholder, and we show "Enter an org alias or use default alias (Press 'Enter' to confirm or 'Escape' to cancel)". The first part is `parameter_gatherer_enter_alias_name`. It doesn't look like we have a message for that latter part, so VS Code must be adding it automatically. (https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts#L63 is where we process `parameter_gatherer_enter_alias_name`.)

Let's do this:

`parameter_gatherer_enter_scratch_org_expiration_days`: 
'Enter the number of days (1–30) until scratch org expiration or use the default value' (followed by the default message about Enter and Escape)

placeholder: 
'7'

Reply:
@ruthemmanuelle @renatoliveira 
The message 'Enter the number of days (1–30) until scratch org expiration or use the default value'  is a bit confusing unless you happen to know what the default is.

'Enter the number of days (1–30) until scratch org expiration or use the default value (7)' may be better here.

Reply:
@renatoliveira @ruthemmanuelle Things look pretty much good to go except the changed to add (7) to the end of the string. Since it's approved by @ruthemmanuelle I'll probably just make that change locally instead of asking for another iteration.